var/spack/repos/builtin/packages/amdlibm/package.py

```diff
@@ -65,7 +66,7 @@ class Amdlibm(SConsPackage):
 
     def build_args(self, spec, prefix):
         """Setting build arguments for amdlibm"""
-        args = ["--prefix={0}".format(prefix)]
+        args = [f"-j{determine_number_of_jobs(parallel=True)}", f"--prefix={prefix}"]
```

Comment:
This change seems unrelated with other changes. Can you move it to a separate PR?

Reply:
New PR raised - amdblibm parallelization build support #43258